src/templates/bbb-enter-name.de.html

```diff
@@ -1,5 +1,7 @@
 <h1>Dem Raum „{{room_name}}“ beitreten</h1>
 
+<p>Nach dem Beitreten wirst Du zu unserem Videokonferenzsystem weitergeleitet, das durch Senfcall betrieben wird, einem gemeinnützigen Projekt, das unseren hohen Datenschutzansprüchen entspricht.</p>
```

Comment:
I think we should also link to their privacy policy or ours? So that people know what they are getting into.

Reply:
But we are linking to ours already, which is the relevant one.

Reply:
Where are we linking to our pp?

Reply:
https://github.com/datenanfragen/backend/blob/5089fb732f171a4593d9632779eb5f4e32ae1188/src/templates/layout/layout.html#L14

Reply:
Well yes, but don’t you think this should be more prominent on this page?

Reply:
How much more prominent than this can it be? :o 

![image](https://github.com/user-attachments/assets/7fff79a3-7dcf-48c5-a75c-91e5bf131cb4)